sharktank/tests/ops/ops_test.py

```diff
@@ -352,6 +352,22 @@ def forward(self, a, d, qs, m):
         self.assertIn("mmt_block_scaled_offset_q4_unsigned.default", s)
 
 
+class TestScatterAdd(unittest.TestCase):
+    def setUp(self):
+        torch.random.manual_seed(0)
+
+    def test(self):
```

Comment:
You should have a test for when `src` is a number. I don't think that the override with match against it.